paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -140,7 +140,7 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
         setupTopContainer()
 
         linkButton.apply {
-            onClick = viewModel::launchLink
+            onClick = { viewModel.launchLink() }
```

Comment:
nit: is this change necessary?

Reply:
Yes, the method now takes an optional parameter, so it needs to be called.